docs/appendices/migration-guides/index.mdx

```diff
@@ -12,6 +12,12 @@ import ContactLink from "@site/src/components/ContactLink";
 import Tabs from "@theme/Tabs";
 import TabItem from "@theme/TabItem";
 
+## `2.18.0` -> `2.19.0`
+
+### New features in `2.18.0`
+
+- [Implement a custom payment method](/docs/advanced/payments/custom-payment-method)
```

Comment:
The documentation on how to do it is indeed new

Reply:
yeah but the _New features_ sections in the Migration guide are for
FC's new features (this was introduced to advertise a bit new features to integrators while they were reading the migration guide for a given release) so that feels a bit odd, there's nothing.

Reply:
@real34 What is your opinion ? You were the one listing the task to do that

Reply:
I don't remember when I noted that.\
I agree with @dpobel that this isn't a feature per se… and no developers might already have done one in their codebase.

We can remove it.

If you mean the initial MR description, it was in a different context so it can be forgotten.
![2022-10-21_18-44](https://user-images.githubusercontent.com/75968/197247165-63f0879c-391e-4c65-a5c7-88d5da39e049.png)